messaging/src/main/java/org/axonframework/eventhandling/EventTrackerStatus.java

```diff
@@ -74,4 +76,21 @@
      * @return the exception that caused processing to fail, or {@code null} when processing normally
      */
     Throwable getError();
+
+    /**
+     * Return the estimated relative replay position this Segment represents.
+     * In case no estimation can be given or no replay is active, an {@code OptionalLong.empty()} will be returned.
```

Comment:
Suggestion: might be nice to mention on this piece of javadoc that this method will return a non-empty result as long as `isReplaying` returns true.